pal/minigame/baidu.ts

```diff
@@ -23,9 +23,9 @@
 */
 
 import { IMiniGame } from 'pal/minigame';
-import { checkPalIntegrity, withImpl } from '../integrity-check';
+import { checkPalIntegrity, withImpl, cloneObject, createInnerAudioContextPolyfill } from '@pal/utils';
 import { Orientation } from '../screen-adapter/enum-type';
-import { cloneObject, createInnerAudioContextPolyfill } from '../utils';
+import { warn } from '../../cocos/core/platform/debug';
```

Comment:
Can import from core instead of `cocos/core/platform/debug`?

Reply:
better not, it is easy to cause circular references

Reply:
yes